src/Instance/InstanceDetails/InstanceAcquisition/InstanceAcquisition.test.js

```diff
@@ -1,17 +1,30 @@
 import React from 'react';
 import { BrowserRouter as Router } from 'react-router-dom';
 
+import { useStripes } from '@folio/stripes/core';
 import { screen } from '@folio/jest-config-stripes/testing-library/react';
 
 import '../../../../test/jest/__mock__';
-import renderWithIntl from '../../../../test/jest/helpers/renderWithIntl';
+import {
+  renderWithIntl,
+  translationsProperties,
+} from '../../../../test/jest/helpers';
 
 import { resultData, line } from './fixtures';
 import InstanceAcquisition from './InstanceAcquisition';
 import useInstanceAcquisition from './useInstanceAcquisition';
 
+import * as utils from '../../../utils';
+
+
+jest.mock('@folio/stripes/core', () => ({
+  ...jest.requireActual('@folio/stripes/core'),
+  useStripes: jest.fn(),
+}));
```

Comment:
It is better to mock `useStripes` [here](https://github.com/folio-org/ui-inventory/blob/e0bc47eea5b2b1c365d5ed9e534a129e61c56615/test/jest/__mock__/stripesCore.mock.js#L86).